src/wasm/literal.cpp

```diff
@@ -93,49 +140,18 @@ Literal::Literal(const Literal& other) : type(other.type) {
       }
     }
   }
-  TODO_SINGLE_COMPOUND(type);
```

Comment:
Was this deleted intentionally?

Reply:
Yes. Before it was guarding the `type.getBasic()` call right after it, but now there is nothing there to guard.

Reply:
I mean, shouldn't we move this with the `switch` too?

Reply:
We already have a check `if (type.isBasic())` right before it, so it is safe to do `type.getBasic()` there, unless I'm missing something.

Reply:
Ah right, sorry for the confusion!